### ISSUE TITLE
Remove sleep statement from DoomEnv render

### ISSUE THREAD

**Issue by gdb** [openai/gym#168]
See: https://github.com/openai/gym/blob/master/gym/envs/doom/doom_env.py#L62

I think we can just remove it without thinking hard, but it's a fair point that the game will be at ridiculous speed otherwise. Maybe we should add an option for a fixed FPS run as a Gym feature. @jietang what do you think?

**Comment by jietang** [comment]
I'm okay with removing it. The game runs at say 75fps instead of 30fps if you don't sleep elsewhere, but you can always do that in agent-land or in the video encoder when writing out a movie.

**Comment by ppaquette** [comment]
I'll take care of this in PR #157

**Comment by olegklimov** [comment]
Shouldn't it be the same `vsync=True` thing? In classic control rendering.py, FPS is equal to monitor refresh rate.

Sure game pace depends on your monitor, but as long as you don't play (as opposed to watch) yourself, it looks pretty natural.

**Comment by ppaquette** [comment]
VizDoom calls freedoom with '+cl_capfps 1 +vid_vsync 0'. The first parameter caps the FPS at 35 to have a constant 35 frames per second (which was the FPS in the original Doom game).  

[VizDoom C++ line here](https://github.com/Marqt/ViZDoom/blob/06ca14e1ab9010f41811d8740d324c2583a175fb/src/lib/ViZDoomController.cpp#L1130)

I've fixed this bug by adding a property 'mode' to the environment which can take 3 values: 'fast', 'normal', 'human'
- 'fast' (default) removes the sleep statement and runs the simulation at roughly 70 fps
- 'normal' keeps the sleep statement and maintains 35 fps
- 'human' launches the environment so a human can try the map (hard to achieve without this setting)

I've also set metadata.video.frames_per_second at 35.

The property would be set as follow:
```python
import gym
env = gym.make('DoomBasic-v0')
env.mode = 'human'
env.reset()
```

This should fix this bug.
